.github/workflows/build-images.yaml

```diff
@@ -66,17 +95,54 @@ jobs:
         id: add-latest-tag
         run: |
           echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Run make bundle
-        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
+      - name: Add release tag
+        if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
+        id: add-branch-tag
+        run: |
+          TAG_NAME=${GITHUB_REF_NAME/\//-}
+          echo "TAG_NAME=${TAG_NAME}" >> $GITHUB_ENV
+          echo "IMG_TAGS=${TAG_NAME} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
+      - name: Set Operator version
+        id: operator-version
+        run: |
+          if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
+          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          else
+          echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
+          fi
+      - name: Add SHA tags if triggered by scheduled workflow
+        if: ${{ github.event_name == "workflow_call" }}
```

Comment:
The conditional steps setup the envs passed to the build bundle, if nothing matches, goes to defaults in the Makefile... or the particular event will set the desired variables. I found it the least verbose, but it could be more explicit if moved to a different job altogether and triggered conditionally. I could do this change if needed or maybe we could elaborate a bit more on your proposition? let me know :)

Reply:
Makes total sense, thanks for the suggestion. Take a look and let me know your thoughts